README.md

```diff
@@ -1,4 +1,11 @@
-node-voicemail-admin
-====================
+# Asterisk Voicemail Administrator Application
 
-A command line administration tool for managing mailboxes in node-voicemail
+A command line administration tool for managing mailboxes in Asterisk voicemail
+
+# Installation
+
+# Usage
+
+```bash
+$ node app.js
+```
```

Comment:
Some example usage might be good here. Even though the tool is good at guiding users, lack of documentation will drive people away.